wp-content/themes/csisnuclearnetwork/inc/site-nav-walker.php

```diff
@@ -0,0 +1,109 @@
+            $author_list . '
+            <div class="site-nav__feat-post-thumbnail"><a href="' . esc_url ( $permalink ) . '"> <img src="' . esc_url ( $thumbnail ) . '"></a></div>';
+          }
+
+        $class_names = $value = '';
+        $classes = empty( $item->classes ) ? array() : (array) $item->classes;
+        $classes[] = 'sub-menu sub-menu-depth-' . $depth;
+        $class_names = join( ' ', apply_filters( 'nav_menu_css_class', array_filter( $classes ), $item, $args ) );
+        $class_names = $class_names ? ' class="' . esc_attr( $class_names ) . '"' : '';
+
+        $indent = str_repeat("\t", $depth);
+        $output .= "\n$indent<div class='sub-menu-container sub-menu-container-depth-" . $depth . " row'>
+            <div class='col-xs-12 col-md sub-menu-children' aria-expanded='false'>\n";
+
+        if ( $args->has_featured ) {
+          $output .= "<div class='hidden-xs col-md-4 sub-menu-featured-post'>
+                      " . $featured_post_html . "
+                      </div>\n";
+        }
+
+        $output .= "<ul" . $class_names . ">\n";
+    }
+
+    function end_lvl( &$output, $depth = 0, $args = array() ) {
+        $indent = str_repeat("\t", $depth);
+        $output .= "$indent</ul>\n
+            </div>\n";
+    }
+
+    function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0 ) {
+        $args->parent_title = $item->title;
+
+        $args->has_featured = false;
+        $args->featured_post = get_field('featured_post_main_menu', $args->menu);
+        
+        if (in_array('site-nav__has-featured', $item->classes)) {
+          $args->has_featured = true;
+        }
+
+        // var_dump($args);
+
+        $indent = ( $depth ) ? str_repeat( "\t", $depth ) : '';
+        $class_names = $value = '';
+        $classes = empty( $item->classes ) ? array() : (array) $item->classes;
+        $classes[] = 'menu-item-' . $item->ID;
+        $class_names = join( ' ', apply_filters( 'nav_menu_css_class', array_filter( $classes ), $item, $args ) );
+        $class_names = $class_names ? ' class="' . esc_attr( $class_names ) . '"' : '';
+        $id = apply_filters( 'nav_menu_item_id', 'menu-item-'. $item->ID, $item, $args );
+        $id = $id ? ' id="' . esc_attr( $id ) . '"' : '';
+        $output .= $indent . '<li' . $id . $class_names . '>';
+        $attributes  = ! empty( $item->attr_title ) ? ' title="'  . esc_attr( $item->attr_title ) .'"' : '';
+        $attributes .= ! empty( $item->target )     ? ' target="' . esc_attr( $item->target     ) .'"' : '';
+        $attributes .= ! empty( $item->xfn )        ? ' rel="'    . esc_attr( $item->xfn        ) .'"' : '';
+        $attributes .= ! empty( $item->url )        ? ' href="'   . esc_attr( $item->url        ) .'"' : '';
+        $item_output = $args->before;
+        if ( $args->walker->has_children && $depth == 0 ) {
+          $item_output .= '<button>' . $attributes;
```

Comment:
I would recommend removing the `$attributes` from the instances where you're using `<button>`. If you look at what the attributes are (defined on lines 86-89), they're pretty link specific, so we can remove them from here.